packages/azure-http-specs/specs/azure/resource-manager/operation-templates/lro.tsp

```diff
@@ -0,0 +1,203 @@
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    
+    Expected verb: GET
+    Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao
+    
+    Expected status code: 200
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao",
+      "name": "aao",
+      "status" : "InProgress"
+    }
+    ```
+    
+    Expected verb: GET
+    Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao
+    
+    Expected status code: 200
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao",
+      "name": "aao",
+      "status" : "Succeeded"
+    }
+    ```
+    
+    (The last GET call on resource URL is optional)
+    Expected verb: GET
+    Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro
+    
+    Expected status code: 200
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro",
+      "name": "top",
+      "type": "topLevel",
+      "location": "eastus",
+      "properties": {
+        "description": "valid",
+        "provisioningState": "Succeeded"
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    """)
+  createOrReplace is ArmResourceCreateOrReplaceAsync<LroResource>;
+
+  #suppress "deprecated" "legacy"
+  #suppress "@azure-tools/typespec-azure-resource-manager/lro-location-header" "legacy"
+  @scenario
+  @scenarioDoc("""
+    Resource DELETE operation.
+    Service returns both Location and Azure-AsyncOperation header on initial request.
+    final-state-via: location
+    
+    Expected verb: DELETE
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected response status code: 202
+    Expected response headers: 
+      - Azure-AsyncOperation={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_delete/aao
+      - Location={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/lro_delete/location
```

Comment:
Just confirm: since the `final-state-via` is `location` in this case, Python SDK will make a final GET for url from `location`, what's the resonponse for it? It is not declared in scenarioDoc.

Reply:
Here: https://github.com/Azure/typespec-azure/pull/1821/files#diff-acc7bdbf75c64c622679dfccf6922b83b1512d07a5ce265894d681140d53e4daR151-R168

When you do GET `location` 1st time, it'll respond with
```markdown
Expected status code: 202
Expected response body:
{
  "id": "location",
  "status": "InProgress"
}
```

2nd and on will be
```markdown 
Expected status code: 204
No response body.
```

Reply:
Added `AAO+Location` for POST. Changed DELETE to only use `Location`.